src/pages/plugins/index.hbs

```diff
@@ -30,17 +32,47 @@ class: plugins
 
     <div class="panel">
       <div class="card">
-        <h3><a href="https://github.com/Esri/esri-leaflet-vector/">Vector Tiles</a></h3>
+        <h3><a href="https://github.com/Esri/esri-leaflet-clustered-feature-layer">Clustered Feature Layer</a></h3>
+        <p>
+          Visualize feature layers using clustering with the help of <a href="https://github.com/Leaflet/Leaflet.markercluster">Leaflet.markercluster</a>.
+        </p>
+        <iframe src="https://ghbtns.com/github-btn.html?user=esri&repo=esri-leaflet-cluster&type=fork&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
+        <br>
+        <iframe src="https://ghbtns.com/github-btn.html?user=esri&repo=esri-leaflet-cluster&type=star&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
+      </div>
+    </div>
+  </div>
+  
+  <div class="row clearfix">
+    <div class="panel">
+      <div class="card">
+        <h3><a href="https://github.com/Esri/esri-leaflet-heatmap-feature-layer">Heatmap Feature Layer</a></h3>
+        <p>
+          Visualize feature layers with heatmaps via <a href="https://github.com/Leaflet/Leaflet.heat">Leaflet.heat</a>.
+        </p>
+        <iframe src="https://ghbtns.com/github-btn.html?user=esri&repo=esri-leaflet-heatmap&type=fork&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
+        <br>
+        <iframe src="https://ghbtns.com/github-btn.html?user=esri&repo=esri-leaflet-heatmap&type=star&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
+      </div>
+    </div>
+
+    <div class="panel">
+      <div class="card">
+        <h3><a href="https://github.com/Esri/Leaflet.shapeMarkers">shapeMarkers</a></h3>
         <p>
-          An <b>experimental</b> plugin to visualize Vector tiles from ArcGIS Online, built on top of <a href="https://github.com/mapbox/mapbox-gl-leaflet">mapbox-gl-leaflet</a>.
+          Generic, fixed size, shape marker symbols for points in Leaflet.
         </p>
-        <iframe src="https://ghbtns.com/github-btn.html?user=esri&repo=esri-leaflet-vector&type=fork&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
+        <iframe src="https://ghbtns.com/github-btn.html?user=Esri&repo=Leaflet.shapeMarkers&type=fork&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
         <br>
-        <iframe src="https://ghbtns.com/github-btn.html?user=esri&repo=esri-leaflet-vector&type=star&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
+        <iframe src="https://ghbtns.com/github-btn.html?user=Esri&repo=Leaflet.shapeMarkers&type=star&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
       </div>
     </div>
+
   </div>
 
+
+  <h3>Community</h3>
+
```

Comment:
```suggestion

  <p>
    Plugins created and generously shared by users and third-party developers of the Esri Leaflet ecosystem.
  </p>

```

Here's an idea for a "subtitle" of sorts to explain what is meant by "Community"; however, I don't want the word "third-party" to feel othering or unnecessarily draw borders here.  I'm open to feedback and wording adjustments.